jena-base/src/main/java/org/apache/jena/atlas/lib/ListUtils.java

```diff
@@ -34,49 +34,40 @@
 {
     private ListUtils() {}
 
-    public static <T>
-    List<T> unique(List<T> list)
-    {
-    		return toList(list.stream().distinct()) ;
+    public static <T> List<T> unique(List<T> list) {
+        return toList(list.stream().distinct());
     }
-    
-    public static
-    List<Integer> asList(int... values)
-    {
-        List<Integer> x = new ArrayList<>(values.length) ;
+
+    public static List<Integer> asList(int...values) {
```

Comment:
Couldn't we replace this method by something like `Arrays.asList(1, 2, 3);` ? Or maybe deprecate it?

Reply:
The signature of `Arrays.asList` is  `List<T> asList(T... a)` and auto-boxing does not work when the argument is `int[]` or `int ...`).

Reply:
Oh, my mistake :) :+1: